packages/table/src/common/_loading.scss

```diff
@@ -14,25 +14,29 @@
   }
 }
 
-.bp-table-cell,
-.bp-table-header {
-  &.pt-loading {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    cursor: default;
+@mixin hide-text() {
+  // Always set color to transparent so cell content can fade-in when transitioning from the
+  // loading state
+  // stylelint-disable-next-line declaration-no-important
+  color: transparent !important;
+}
 
-    // Always set color to transparent so cell content can fade-in when transitioning from the
-    // loading state
-    // stylelint-disable-next-line declaration-no-important
-    color: transparent !important;
-    pointer-events: none;
-  }
+@mixin disable-interaction() {
+  cursor: default;
+  pointer-events: none;
+}
+
+@mixin vertical-center() {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
 
+@mixin table-skeleton($skeleton-height) {
   .pt-skeleton {
     opacity: 0;
-    height: $pt-grid-size / 2;
+    height: $skeleton-height;
     animation: $cell-transition-duration linear forwards skeleton-fade-in, $skeleton-animation;
-    animation-delay: 0, $cell-transition-duration;
+    animation-delay: 0s, $cell-transition-duration;
```

Comment:
this syntax doesn't seem to fully work in firefox for some reason: the fade in occurs but the glow animation never happens. perhaps an implementation bug on their end?